emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -229,12 +224,22 @@ public void dismiss() {
     recentEmoji.persist();
     variantEmoji.persist();
 
+    emojiResultReceiver.setReceiver(null);
+
     if (originalImeOptions != -1) {
       editText.setImeOptions(originalImeOptions);
       final InputMethodManager inputMethodManager = (InputMethodManager) context.getSystemService(Context.INPUT_METHOD_SERVICE);
+
       if (inputMethodManager != null) {
         inputMethodManager.restartInput(editText);
       }
+
+      if (SDK_INT >= O) {
+        AutofillManager autofillManager = context.getSystemService(AutofillManager.class);
+        if (autofillManager != null) {
+          autofillManager.cancel();
```

Comment:
Great! I sometimes saw this leak in my app, but always thought that was an Android SDK one.

Reply:
:) It's almost like magic! :)

Reply:
I'm pretty sure this is the only available open source lib these days that handles keyboard absolutely correctly btw, which is a great accomplishment - thanks to both @rubengees and @vanniktech for pushing when things didn't work as they were supposed to!